client/blocks/plan-storage/test/bar.jsx

```diff
@@ -1,3 +1,7 @@
+/**
+ * @jest-environment jsdom
```

Comment:
Should we be setting this globally? Or does it need to be in every file?

Reply:
I say let's keep this local for now. We could enable it globally once we get rid of `enzyme` completely.

Reply:
I generally do see folks just carrying it around from file to file as well. Not all tests will be using `testing-library`/`jsdom`, so it’s kind of a signal that the environment is actually needed for the proceeding tests.